### PR TITLE
Temporarly disable Travis CI  Mac OS X

### PR DESCRIPTION
**Pull request by G10h4ck**

### CODE REVIEW

**Comment by G10h4ck** [comment]
Travis CI has updated to Mac OS X 10.13 the new compiler complain a lot about RetroShare code...

**Comment by G10h4ck** [comment]
this may be related https://github.com/termux/termux-packages/issues/1149

File: retroshare.pri [hunk 1]
```diff
@@ -500,6 +501,12 @@ macx-* {
 		QMAKE_CFLAGS += -Wno-nullability-completeness
 	}
 
+    rs_macos10.13 {
+        message(***retroshare.pri: Set Target and SDK to MacOS 10.12 )
```
Comment (sehraf) [inline retroshare.pri]: nit: 13 instead of 12

**Comment by PhenomRetroShare** [comment]
Already worked on it with https://github.com/PhenomRetroShare/RetroShare/tree/Fix_TravisMacOSCompil. But get blocked to Compil error with unordered_map:
https://travis-ci.org/PhenomRetroShare/RetroShare/jobs/411227842#L1340

**Comment by G10h4ck** [comment]
yeah it's same error here also i have found more probably related reports on other projects

https://github.com/sass/libsass-python/issues/240

**Pull request closed without merging**